scripts-dev/lint.sh

```diff
@@ -94,3 +94,4 @@ isort "${files[@]}"
 python3 -m black "${files[@]}"
 ./scripts-dev/config-lint.sh
 flake8 "${files[@]}"
+mypy
```

Comment:
Note that I think running this over `$files[@]` doesn't make sense until we support mypy in all of our files.